config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  EXPIRATION_TIME_FOR_WEB_TOKEN: '2h'
```

Comment:
I think would be nice if you move this to some environment value and load here on the config.js every environment value that you want, in that way you will have all the usage of all the environmental values in just one place and also you may have default values

```js
module.exports = {
  EXPIRATION_TIME_FOR_WEB_TOKEN: process.env.EXPIRATION_TIME_FOR_WEB_TOKEN || '2h',
  SECRET: process.env.SECRET || 'the default secret',
  SALT_DEFAULT_WORK_FACTOR: JSON.parse(process.env.SALT_DEFAULT_WORK_FACTOR || '10')
  // ...
};
```

Reply:
Done!